src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewCell.swift

```diff
@@ -122,6 +123,24 @@ extension DynamicCell {
 			cell.backgroundColor = color
 		}
 	}
+	
```

Comment:
We already have an almost identical bullet point-esque cell, which can be found in `ExposureSubmissionDynamicCell.swift` in the method 

``` swift
static func stepCell(bulletPoint title: String, hairline: ExposureSubmissionStepCell.Hairline = .none, bottomSpacing: ExposureSubmissionStepCell.Spacing = .normal, action: DynamicAction = .none) -> DynamicCell
```

Your new bullet cell class is awesome and it is the way to go. However, due to consistency reasons, it would be fantastic if you changed the occurrences of the 'old' bullet point cells to support your new bullet cell.

Reply:
Hmm is it really the same kind of cell though? `stepCell` seems to be much more complex. For example it has an `Image`, `hairline` and `spacing` that can be set. Feels like this would make it much to complex to have all of this different styles in one cell.

Reply:
Ahh I think I misunderstood you the first time. Yeah totally agree, we removed `static func stepCell(bulletPoint title: String, hairline: ExposureSubmissionStepCell.Hairline = .none, bottomSpacing: ExposureSubmissionStepCell.Spacing = .normal, action: DynamicAction = .none) -> DynamicCell
` and replaced the occurrences with `.bulletPoint` https://github.com/corona-warn-app/cwa-app-ios/pull/1145/commits/fdad69c3fe1dabf76bf0a9ce1e6ebfd321b7099b

Reply:
That sounds lovely! Thanks for adjusting, really appreciate that 👍